collector/receiver/telemetryapireceiver/README.md

```diff
@@ -0,0 +1,26 @@
+# Telemetry API Receiver
+
+| Status                   |                 |
+| ------------------------ |-----------------|
+| Stability                | [in development]|
+| Supported pipeline types | traces          |
+| Distributions            | [extension]     |
+
+This receiver generate telemetry in response to events from the [Telemetry API](https://docs.aws.amazon.com/lambda/latest/dg/telemetry-api.html). It does this by setting up an endpoint, and registering itself with the Telemetry API on startup.
+
+Supported events:
+
+* `platform.initStart`
+* `platform.initRuntimeDone`
```

Comment:
It's not clear to me from this documentation what the receiver will do with these two events.

Reply:
Updated documentation, PTAL